lua/entities/gmod_wire_expression2/core/custom/cl_playercore.lua

```diff
@@ -29,8 +29,8 @@ net.Receive("wire_expression2_playercore_sendmessage", function( len, ply )
         -- printColorDriver is used for the first time on us by this chip
         WireLib.AddNotify(msg1, NOTIFY_GENERIC, 7, NOTIFYSOUND_DRIP3)
         WireLib.AddNotify(msg2, NOTIFY_GENERIC, 7)
-        chat.AddText(Color(255, 50, 50),"After this message, ", ply, " can send you a 100% realistically fake people talking, including admins.")
-        chat.AddText(Color(255, 50, 50),"Look the console to see if the message is from an expression2")
+        chat.AddText(Color(255, 50, 50), ply, "is using E2 sendmessage." )
+        chat.AddText(Color(255, 50, 50), "Every message that starts with a 'RED >' is from an expression2" )
```

Comment:
this isn't true is it?

Reply:
![image](https://github.com/user-attachments/assets/8773d492-e20a-424a-ae73-aa019e45911a)
here's an example, can see the red >

Reply:
Source: https://github.com/CFC-Servers/playercore/blob/master/lua/entities/gmod_wire_expression2/core/custom/cl_playercore.lua#L37